### PR TITLE
Cohort panel cnv calling

### PR DESCRIPTION
**Pull request by TammushB**

### CODE REVIEW

File: Dockerfile.jbvc [hunk 1]
```diff
@@ -13,6 +13,14 @@ RUN conda env create -f /VariantCalling/setup/environment.yml && \
     conda env create -f /VariantCalling/setup/other_envs/cnmops.yml && \
     conda init --all
 
+## install ug-cn.mops:
+RUN cd /opt && git clone --branch ilyasoifer/test.1.40 https://github.com/Ultimagen/cn.mops.git cn.mops
+
+RUN source ~/.bashrc && \
+   conda activate cn.mops && \
+   Rscript -e 'remove.packages("cn.mops")' && \
```
Comment (ilyasoifer) [inline Dockerfile.jbvc:21]: Cleaner to remove cn.mops from the environment I think

File: ugvc/cnv/create_reads_count_cohort_matrix.R [hunk 2]
```diff
@@ -0,0 +1,50 @@
+suppressPackageStartupMessages(library(cn.mops))
```
Comment (ilyasoifer) [inline ugvc/cnv/create_reads_count_cohort_matrix.R:18]: Can you please add the license header to this script?

**Review by ilyasoifer — APPROVED** [review]
Looks good, just minor things
One more request - can you update the README of the package to mention the cn.mops environment as part of the install?

File: Dockerfile.jbvc (hunk 1 above)
Comment (TammushB) [reply Dockerfile.jbvc:21]: I don't agree. 
the conda environment should include some version of cn.mops. 
We decide to override the installation with another/dev version.  
I am not sure, but maybe we can export the conda environment after installation...?

File: ugvc/cnv/create_reads_count_cohort_matrix.R (hunk 2 above)
Comment (TammushB) [reply ugvc/cnv/create_reads_count_cohort_matrix.R:18]: Do you think we need to add the license header to the following scripts as well? : 
ugvc/cnv/get_reads_count_from_bam.R
ugvc/cnv/merge_reads_count_sample_to_cohort.R

**Pull request merged**